docs/source/models.rst

```diff
@@ -3022,6 +3022,25 @@ This input format requires that both the bitrate has been set prior to conversio
     audio transformers pipelines being set as expecting ``binary`` (``bytes``) data. The serving endpoint cannot accept a union of types, so a particular model instance must choose one
     or the other as an allowed input type.
 
+SentenceTransformers (``sentence_transformers``) (Experimental)
```

Comment:
@BenWilson2 when do we move flavors from experimental to non-experimental?

Reply:
Typically after 2 minor releases have had no modifications to the flavor or the boundary of a major release.